### ISSUE TITLE
khal UI mockups

### ISSUE THREAD

**Issue by Rostiger** [pimutils/khal#990]
Hey all! I've been using khal for a while and really appreciate the simplicity and focus of it.
However, I'm using quite a few calendars and noticed that I'm having trouble with the readability of certain elements, leading up to missing a real life appointment, because my eyes just skipped over it!

Having done a lot of UI design in my past, this got me thinking on how I would adapt the UI if I could change it any way I wanted. In a surge of motivation, I started creating mockups, which I'd like to share here now. Maybe some of the ideas resonate and even might be implemented (I consider my own coding skills still too inferior to have the guts of actually diving in an submitting PRs)!

Anyway, I'm happy for any thoughts and feedback on this.
Cheers!
![khal_ui](https://user-images.githubusercontent.com/661698/102005092-6ea7d600-3d16-11eb-83e5-d38f018ac860.png)

**Comment by Tonus1** [comment]
Looks great

**Comment by ephase** [comment]
Hi @Rostiger, this looks really great but there is lot of vertical space lost. `ikhal` is really usable in little terminal window, mine is 12x90 for example, as it should be in my opinion.

I love the horizontal rhythm of your mockups, I'll try to implement your events list proposal  in a prototype (but in my actual knowledge of urwid limit me for now)

Thank you.

**Comment by d7415** [comment]
I also think it's important to consider normal `khal` when changing the UI - I don't think they should diverge any more than necessary.

**Comment by frankgerhardt** [comment]
I like the idea :-)

**Comment by AtomToast** [comment]
looks great

**Comment by Rostiger** [comment]
Whoops, got distracted and forgot all about replying here!

> Hi @Rostiger, this looks really great but there is lot of vertical space lost. `ikhal` is really usable in little terminal window, mine is 12x90 for example, as it should be in my opinion.

It's true that it takes up a lot more vertical space. The idea was to separate days more to prevent mismatching days and events. It happens to me A LOT that I think events are on the current day, when really they are on the next, but the current one doesn't have any events: 
![image](https://user-images.githubusercontent.com/661698/148729259-32b3762a-e838-4408-a4ba-e1ba5dbd9abc.png)

The better solution might be to add a config option to only show one day instead of two in the view pane?
I'll try to come up with a solution that works for smaller terminal sizes as well.

> I also think it's important to consider normal khal when changing the UI - I don't think they should diverge any more than necessary.
Good point! I didn't consider khal as I'm only using ikhal. I'll include it in the revision.

**Comment by AtomToast** [comment]
It may be possible to just have a compact mode that automatically engages on smaller sizes or can be set as a preference in the settings. You could also have it so days have an alternating Background color so it's easier to tell what belongs to what day?

**Comment by Rostiger** [comment]
Yeah, these are all valid options and should be preferred to re-designing the UI for more vertical space.
